main.py

```diff
@@ -566,7 +566,7 @@ def main(args):
                         help='Output file path. If the folder does not exist, it will be created.',
                         type=telescope.utils.create_directory_if_not_exists)
   parser.add_argument('--maxminddir', default='resources/', help='MaxMind GeoLite ASN snapshot directory.')
-  parser.add_argument('--savequery', default=False, action='store_true',
+  parser.add_argument('--savequery', default=True, action='store_true',
```

Comment:
I think it makes sense as a command line switch, but I don't think the default should be to save the query because that's not what we expect the user to be primarily interested in. @collina - care to weigh in?